tests/comfy_cli/command/test_node_deps.py

```
@@ -314,6 +325,534 @@ def test_no_names_reports_every_pack(workspace, fake_pip):
+
+    with mock.patch.object(RegistryAPI, "get_node", autospec=True) as get_node:
+        get_node.return_value = _node("bare-pack", "1.0.0", [])
+        node_deps_cmd.execute(r, str(workspace), registry_ids=["bare-pack"])
+
+    out = capsys.readouterr().out
+    assert "bare-pack" in out
+    assert "did not return dependency metadata" in out
+    assert r.exit_code == 0  # a report, not a check
+
+
+def test_registry_flags_are_registered_on_the_deps_command():
```

Pylint (4.0.6)

[convention] Missing function or method docstring

(C0116)